docs/index.md

```diff
@@ -246,7 +281,7 @@ maxLedgerVersion | integer | *Optional* The highest ledger version that the tran
 maxLedgerVersionOffset | integer | *Optional* Offset from current legder version to highest ledger version that the transaction can be included in.
 sequence | [sequence](#account-sequence-number) | *Optional* The initiating account's sequence number for this transaction.
 
-We recommended that you specify a `maxLedgerVersion` because without it there is no way to know that a failed transaction will never succeeed in the future. It is impossible for a transaction to succeed after the network ledger version exceeds the transaction's `maxLedgerVersion`.
+We recommended that you specify a `maxLedgerVersion` so that you can quickly determine that a failed transaction will never succeeed in the future. It is impossible for a transaction to succeed after the network ledger version exceeds the transaction's `maxLedgerVersion`. If you do not specify a `maxLedgerVersion`, a default will be used (three greater than the current ledger version), which can be found in the return value to the "prepare" method.
```

Comment:
Suggested rephrase of the last paragraph:

If you omit `maxLedgerVersion`, the prepare* method automatically supplies a `maxLedgerVersion` equal to the current ledger plus 3, which it includes in the return value from the prepare* method.